testutil/src/test/java/org/spine3/test/VerifyShould.java

```diff
@@ -0,0 +1,1312 @@
+    }
+
+    @Test
+    public void pass_if_map_is_empty() {
+        Verify.assertEmpty(Collections.emptyMap());
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_multimap_is_not_empty() {
+        final Multimap<Integer, Integer> multimap = ArrayListMultimap.create();
+
+        multimap.put(1, 1);
+
+        Verify.assertEmpty(multimap);
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_multimap_is_null_in_assert_empty() {
+        Verify.assertEmpty((Multimap) null);
+    }
+
+    @Test
+    public void pass_if_multimap_is_empty() {
+        Verify.assertEmpty(ArrayListMultimap.create());
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_iterable_is_empty() {
+        Verify.assertNotEmpty(FluentIterable.of());
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_iterable_is_null_in_iterable_not_empty() {
+        Verify.assertIterableNotEmpty(null);
+    }
+
+    @Test
+    public void pass_if_iterable_is_not_empty() {
+        Verify.assertNotEmpty(FluentIterable.of(1));
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_map_is_empty() {
+        Verify.assertNotEmpty(Collections.emptyMap());
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_map_is_null_in_not_empty() {
+        Verify.assertNotEmpty((Map) null);
+    }
+
+    @Test
+    public void pass_if_map_is_not_empty() {
+        Verify.assertNotEmpty(Collections.singletonMap(1, 1));
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_multimap_is_empty() {
+        Verify.assertNotEmpty(ArrayListMultimap.create());
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_multimap_is_null_in_assert_not_empty() {
+        Verify.assertNotEmpty((Multimap) null);
+    }
+
+    @Test
+    public void pass_if_multimap_is_not_empty() {
+        final Multimap<Integer, Integer> multimap = ArrayListMultimap.create();
+
+        multimap.put(1, 1);
+
+        Verify.assertNotEmpty(multimap);
+    }
+
+    @SuppressWarnings("ZeroLengthArrayAllocation")
+    @Test(expected = AssertionError.class)
+    public void fail_if_array_is_empty() {
+        Verify.assertNotEmpty(new Integer[0]);
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_array_is_null_in_assert_not_empty() {
+        Verify.assertNotEmpty((Integer[]) null);
+    }
+
+    @Test
+    public void pass_if_array_is_not_empty() {
+        final Integer[] array = {1, 2, 3};
+        Verify.assertNotEmpty(array);
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_object_array_size_is_not_equal() {
+        Verify.assertSize(-1, new Object[1]);
```

Comment:
Please use a realistic array size value instead of `-1`. That's a common issue in this PR, so apply the fix to every similar spot.